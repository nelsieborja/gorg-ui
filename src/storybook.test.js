// Snapshot testing
import initStoryshots from '@storybook/addon-storyshots';
initStoryshots();
