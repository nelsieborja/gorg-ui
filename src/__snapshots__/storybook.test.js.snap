// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button Icon Live Props 1`] = `
<div
  className="css-1tba8mc"
>
  <button
    className="css-14kjfrk"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    <svg>
      home.svg
    </svg>
  </button>
  <button
    className="css-kaa8vz"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    <svg>
      home.svg
    </svg>
  </button>
  <button
    className="css-2wssdb"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    <svg>
      home.svg
    </svg>
  </button>
  <button
    className="css-60h0gg"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="rounded"
  >
    <svg>
      calendar.svg
    </svg>
  </button>
  <button
    className="css-1w00dg9"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="rounded"
  >
    <svg>
      calendar.svg
    </svg>
  </button>
  <button
    className="css-cw7lnu"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="rounded"
  >
    <svg>
      calendar.svg
    </svg>
  </button>
  <button
    className="css-wlwewr"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="circle"
  >
    <svg>
      dashboard.svg
    </svg>
  </button>
  <button
    className="css-oz45h3"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="circle"
  >
    <svg>
      dashboard.svg
    </svg>
  </button>
  <button
    className="css-ghjcwx"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="circle"
  >
    <svg>
      dashboard.svg
    </svg>
  </button>
  <br />
  <button
    className="css-1ygfzzs"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    <svg>
      home.svg
    </svg>
  </button>
  <button
    className="css-1qdtxd2"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    <svg>
      home.svg
    </svg>
  </button>
  <button
    className="css-obby7m"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    <svg>
      home.svg
    </svg>
  </button>
  <button
    className="css-1lmhzau"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="rounded"
  >
    <svg>
      calendar.svg
    </svg>
  </button>
  <button
    className="css-vgo6y8"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="rounded"
  >
    <svg>
      calendar.svg
    </svg>
  </button>
  <button
    className="css-1u8a8d"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="rounded"
  >
    <svg>
      calendar.svg
    </svg>
  </button>
  <button
    className="css-17dfk9z"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="circle"
  >
    <svg>
      dashboard.svg
    </svg>
  </button>
  <button
    className="css-lk4c1x"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="circle"
  >
    <svg>
      dashboard.svg
    </svg>
  </button>
  <button
    className="css-1xq41s1"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="circle"
  >
    <svg>
      dashboard.svg
    </svg>
  </button>
  <br />
  <button
    className="css-4bmhyk"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    <svg>
      home.svg
    </svg>
  </button>
  <button
    className="css-dvt8xl"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    <svg>
      home.svg
    </svg>
  </button>
  <button
    className="css-1jqqcfp"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    <svg>
      home.svg
    </svg>
  </button>
  <button
    className="css-1srt9u5"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="rounded"
  >
    <svg>
      calendar.svg
    </svg>
  </button>
  <button
    className="css-2px5rd"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="rounded"
  >
    <svg>
      calendar.svg
    </svg>
  </button>
  <button
    className="css-17vstv7"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="rounded"
  >
    <svg>
      calendar.svg
    </svg>
  </button>
  <button
    className="css-12uu3er"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="circle"
  >
    <svg>
      dashboard.svg
    </svg>
  </button>
  <button
    className="css-j5r8ec"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="circle"
  >
    <svg>
      dashboard.svg
    </svg>
  </button>
  <button
    className="css-h077nx"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="circle"
  >
    <svg>
      dashboard.svg
    </svg>
  </button>
  <br />
</div>
`;

exports[`Storyshots Button Live Props 1`] = `
<div
  className="css-1tba8mc"
>
  <button
    className="css-153n7r7"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    Change my appearance from 
    <strong>
      "Knobs"
    </strong>
     tab
  </button>
</div>
`;

exports[`Storyshots Button.Color Default 1`] = `
<div
  className="css-1tba8mc"
>
  <button
    className="css-153n7r7"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    DEFAULT
  </button>
  <button
    className="css-1xw5j27"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    OUTLINE
  </button>
  <button
    className="css-1kkgwbg"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    TEXT
  </button>
</div>
`;

exports[`Storyshots Button.Color Primary 1`] = `
<div
  className="css-1tba8mc"
>
  <button
    className="css-1lq15mw"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    DEFAULT
  </button>
  <button
    className="css-14o5sa6"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    OUTLINE
  </button>
  <button
    className="css-jcw9dv"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    TEXT
  </button>
</div>
`;

exports[`Storyshots Button.Color Secondary 1`] = `
<div
  className="css-1tba8mc"
>
  <button
    className="css-boshfe"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    DEFAULT
  </button>
  <button
    className="css-1qzj5ml"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    OUTLINE
  </button>
  <button
    className="css-15tu6ha"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    TEXT
  </button>
</div>
`;

exports[`Storyshots Button.Shape Circle 1`] = `
<div
  className="css-1tba8mc"
>
  <button
    className="css-1w51cmx"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="circle"
  >
    DEFAULT
  </button>
  <button
    className="css-1ovk79p"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="circle"
  >
    PRIMARY
  </button>
  <button
    className="css-gq9y2o"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="circle"
  >
    SECONDARY
  </button>
</div>
`;

exports[`Storyshots Button.Shape Default 1`] = `
<div
  className="css-1tba8mc"
>
  <button
    className="css-153n7r7"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    DEFAULT
  </button>
  <button
    className="css-1lq15mw"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    PRIMARY
  </button>
  <button
    className="css-boshfe"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    SECONDARY
  </button>
</div>
`;

exports[`Storyshots Button.Shape Rounded 1`] = `
<div
  className="css-1tba8mc"
>
  <button
    className="css-1omhqi2"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="rounded"
  >
    DEFAULT
  </button>
  <button
    className="css-vnn5kn"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="rounded"
  >
    PRIMARY
  </button>
  <button
    className="css-1jkolc3"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="rounded"
  >
    SECONDARY
  </button>
</div>
`;

exports[`Storyshots Button.State Disabled 1`] = `
<div
  className="css-1tba8mc"
>
  <button
    className="css-153n7r7"
    color="default"
    disabled={true}
    onClick={[Function]}
    shape="default"
  >
    DEFAULT
  </button>
  <button
    className="css-1lq15mw"
    color="primary"
    disabled={true}
    onClick={[Function]}
    shape="default"
  >
    PRIMARY
  </button>
  <button
    className="css-boshfe"
    color="secondary"
    disabled={true}
    onClick={[Function]}
    shape="default"
  >
    SECONDARY
  </button>
  <br />
  <button
    className="css-1xw5j27"
    color="default"
    disabled={true}
    onClick={[Function]}
    shape="default"
  >
    DEFAULT
  </button>
  <button
    className="css-14o5sa6"
    color="primary"
    disabled={true}
    onClick={[Function]}
    shape="default"
  >
    PRIMARY
  </button>
  <button
    className="css-1qzj5ml"
    color="secondary"
    disabled={true}
    onClick={[Function]}
    shape="default"
  >
    SECONDARY
  </button>
  <br />
  <button
    className="css-1kkgwbg"
    color="default"
    disabled={true}
    onClick={[Function]}
    shape="default"
  >
    DEFAULT
  </button>
  <button
    className="css-jcw9dv"
    color="primary"
    disabled={true}
    onClick={[Function]}
    shape="default"
  >
    PRIMARY
  </button>
  <button
    className="css-15tu6ha"
    color="secondary"
    disabled={true}
    onClick={[Function]}
    shape="default"
  >
    SECONDARY
  </button>
  <br />
</div>
`;

exports[`Storyshots Button.State Loading 1`] = `
<div
  className="css-1tba8mc"
>
  <button
    className="css-tz91p6"
    color="default"
    disabled={false}
    shape="default"
  >
    <svg
      className="-loader"
    >
      loader.svg
    </svg>
    DEFAULT
  </button>
  <button
    className="css-1bxn4hu"
    color="primary"
    disabled={false}
    shape="default"
  >
    <svg
      className="-loader"
    >
      loader.svg
    </svg>
    PRIMARY
  </button>
  <button
    className="css-bu6a4x"
    color="secondary"
    disabled={false}
    shape="default"
  >
    <svg
      className="-loader"
    >
      loader.svg
    </svg>
    SECONDARY
  </button>
  <br />
  <button
    className="css-urc6fs"
    color="default"
    disabled={false}
    shape="default"
  >
    <svg
      className="-loader"
    >
      loader.svg
    </svg>
    DEFAULT
  </button>
  <button
    className="css-jurg9a"
    color="primary"
    disabled={false}
    shape="default"
  >
    <svg
      className="-loader"
    >
      loader.svg
    </svg>
    PRIMARY
  </button>
  <button
    className="css-1833d7i"
    color="secondary"
    disabled={false}
    shape="default"
  >
    <svg
      className="-loader"
    >
      loader.svg
    </svg>
    SECONDARY
  </button>
  <br />
  <button
    className="css-1fqvk5w"
    color="default"
    disabled={false}
    shape="default"
  >
    <svg
      className="-loader"
    >
      loader.svg
    </svg>
    DEFAULT
  </button>
  <button
    className="css-1rynzxc"
    color="primary"
    disabled={false}
    shape="default"
  >
    <svg
      className="-loader"
    >
      loader.svg
    </svg>
    PRIMARY
  </button>
  <button
    className="css-1qhv4d2"
    color="secondary"
    disabled={false}
    shape="default"
  >
    <svg
      className="-loader"
    >
      loader.svg
    </svg>
    SECONDARY
  </button>
  <br />
</div>
`;

exports[`Storyshots Button.Variant Default 1`] = `
<div
  className="css-1tba8mc"
>
  <button
    className="css-153n7r7"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    DEFAULT
  </button>
  <button
    className="css-1lq15mw"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    PRIMARY
  </button>
  <button
    className="css-boshfe"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    SECONDARY
  </button>
</div>
`;

exports[`Storyshots Button.Variant Outline 1`] = `
<div
  className="css-1tba8mc"
>
  <button
    className="css-1xw5j27"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    DEFAULT
  </button>
  <button
    className="css-14o5sa6"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    PRIMARY
  </button>
  <button
    className="css-1qzj5ml"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    SECONDARY
  </button>
</div>
`;

exports[`Storyshots Button.Variant Text 1`] = `
<div
  className="css-1tba8mc"
>
  <button
    className="css-1kkgwbg"
    color="default"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    DEFAULT
  </button>
  <button
    className="css-jcw9dv"
    color="primary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    PRIMARY
  </button>
  <button
    className="css-15tu6ha"
    color="secondary"
    disabled={false}
    onClick={[Function]}
    shape="default"
  >
    SECONDARY
  </button>
</div>
`;

exports[`Storyshots Checkbox Live Props 1`] = `
<div
  className="css-jyjmas"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={false}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-pd3ieb"
      color="default"
      shape="default"
    />
    Change my appearance from 
    <strong>
      "Knobs"
    </strong>
     tab
  </label>
</div>
`;

exports[`Storyshots Checkbox.Color Default 1`] = `
<div
  className="css-jyjmas"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1c6iy45"
      color="default"
      shape="default"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1nak7jo"
      color="default"
      shape="rounded"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-8g7p8q"
      color="default"
      shape="circle"
    />
  </label>
</div>
`;

exports[`Storyshots Checkbox.Color Primary 1`] = `
<div
  className="css-jyjmas"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-b7zibr"
      color="primary"
      shape="default"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1mb6i4a"
      color="primary"
      shape="rounded"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1kz8v33"
      color="primary"
      shape="circle"
    />
  </label>
</div>
`;

exports[`Storyshots Checkbox.Color Secondary 1`] = `
<div
  className="css-jyjmas"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1mmgaor"
      color="secondary"
      shape="default"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-bkes9n"
      color="secondary"
      shape="rounded"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-y08ue9"
      color="secondary"
      shape="circle"
    />
  </label>
</div>
`;

exports[`Storyshots Checkbox.Shape Circle 1`] = `
<div
  className="css-jyjmas"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-8g7p8q"
      color="default"
      shape="circle"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1kz8v33"
      color="primary"
      shape="circle"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-y08ue9"
      color="secondary"
      shape="circle"
    />
  </label>
</div>
`;

exports[`Storyshots Checkbox.Shape Default 1`] = `
<div
  className="css-jyjmas"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1c6iy45"
      color="default"
      shape="default"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-b7zibr"
      color="primary"
      shape="default"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1mmgaor"
      color="secondary"
      shape="default"
    />
  </label>
</div>
`;

exports[`Storyshots Checkbox.Shape Rounded 1`] = `
<div
  className="css-jyjmas"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1nak7jo"
      color="default"
      shape="rounded"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1mb6i4a"
      color="primary"
      shape="rounded"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-bkes9n"
      color="secondary"
      shape="rounded"
    />
  </label>
</div>
`;

exports[`Storyshots Checkbox.State Controlled 1`] = `
<div
  className="css-jyjmas"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      checked={false}
      className="css-1rnm75w"
      defaultChecked={false}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1c6iy45"
      color="default"
      shape="default"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      checked={false}
      className="css-1rnm75w"
      defaultChecked={false}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-b7zibr"
      color="primary"
      shape="default"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      checked={false}
      className="css-1rnm75w"
      defaultChecked={false}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1mmgaor"
      color="secondary"
      shape="default"
    />
  </label>
</div>
`;

exports[`Storyshots Checkbox.State Disabled 1`] = `
<div
  className="css-jyjmas"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={false}
      disabled={true}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1c6iy45"
      color="default"
      shape="default"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={false}
      disabled={true}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-b7zibr"
      color="primary"
      shape="default"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={false}
      disabled={true}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1mmgaor"
      color="secondary"
      shape="default"
    />
  </label>
</div>
`;

exports[`Storyshots Checkbox.State Uncontrolled 1`] = `
<div
  className="css-jyjmas"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1c6iy45"
      color="default"
      shape="default"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-b7zibr"
      color="primary"
      shape="default"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1mmgaor"
      color="secondary"
      shape="default"
    />
  </label>
</div>
`;

exports[`Storyshots Expansion Panel Live Props 1`] = `
<div
  className="css-14973vb"
>
  <header
    className="css-jgle3q"
  >
    <h2>
      Expansion Title
    </h2>
    <button
      className="css-4bmhyk"
      color="default"
      disabled={false}
      onClick={[Function]}
      shape="default"
    >
      <svg
        className="-minus"
      >
        minus.svg
      </svg>
    </button>
  </header>
  <section
    className="css-17jzza4"
    onTransitionEnd={[Function]}
    style={
      Object {
        "height": 0,
      }
    }
  >
    <div>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
    </div>
  </section>
</div>
`;

exports[`Storyshots Expansion Panel Multi Expansion Panel 1`] = `
<div
  className="css-14973vb"
>
  <header
    className="css-jgle3q"
  >
    <h2>
      Expansion Title
    </h2>
    <button
      className="css-4bmhyk"
      color="default"
      disabled={false}
      onClick={[Function]}
      shape="default"
    >
      <svg
        className="-plus"
      >
        plus.svg
      </svg>
    </button>
  </header>
  <section
    className="css-17jzza4"
    onTransitionEnd={[Function]}
    style={
      Object {
        "height": 0,
      }
    }
  >
    <div>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
    </div>
  </section>
  <header
    className="css-jgle3q"
  >
    <h2>
      Expansion Title
    </h2>
    <button
      className="css-4bmhyk"
      color="default"
      disabled={false}
      onClick={[Function]}
      shape="default"
    >
      <svg
        className="-minus"
      >
        minus.svg
      </svg>
    </button>
  </header>
  <section
    className="css-17jzza4"
    onTransitionEnd={[Function]}
    style={
      Object {
        "height": 0,
      }
    }
  >
    <div>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
    </div>
  </section>
  <header
    className="css-jgle3q"
  >
    <h2>
      Expansion Title
    </h2>
    <button
      className="css-4bmhyk"
      color="default"
      disabled={false}
      onClick={[Function]}
      shape="default"
    >
      <svg
        className="-plus"
      >
        plus.svg
      </svg>
    </button>
  </header>
  <section
    className="css-17jzza4"
    onTransitionEnd={[Function]}
    style={
      Object {
        "height": 0,
      }
    }
  >
    <div>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
    </div>
  </section>
</div>
`;

exports[`Storyshots Radio Live Props 1`] = `
<div
  className="css-jyjmas"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      disabled={false}
      onChange={[Function]}
      type="radio"
    />
    <i
      className="css-1mije5k"
      color="default"
      shape="circle"
    />
    Change my appearance from 
    <strong>
      "Knobs"
    </strong>
     tab
  </label>
</div>
`;

exports[`Storyshots Radio.Color Default 1`] = `
<div
  className="css-jyjmas"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      disabled={false}
      name="radio"
      onChange={[Function]}
      type="radio"
    />
    <i
      className="css-bksfa4"
      color="default"
      shape="circle"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      disabled={false}
      name="radio"
      onChange={[Function]}
      type="radio"
    />
    <i
      className="css-17ap9mr"
      color="primary"
      shape="circle"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      disabled={false}
      name="radio"
      onChange={[Function]}
      type="radio"
    />
    <i
      className="css-i9ivch"
      color="secondary"
      shape="circle"
    />
  </label>
</div>
`;

exports[`Storyshots Switch Live Props 1`] = `
<div
  className="css-14973vb"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={false}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-1fos6kk"
      color="default"
    />
    Change my appearance from 
    <strong>
      "Knobs"
    </strong>
     tab
  </label>
</div>
`;

exports[`Storyshots Switch.State Controlled 1`] = `
<div
  className="css-jyjmas"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      checked={false}
      className="css-1rnm75w"
      defaultChecked={false}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-umqk88"
      color="default"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      checked={false}
      className="css-1rnm75w"
      defaultChecked={false}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-h1wp49"
      color="primary"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      checked={false}
      className="css-1rnm75w"
      defaultChecked={false}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-49iodi"
      color="secondary"
    />
  </label>
</div>
`;

exports[`Storyshots Switch.State Disabled 1`] = `
<div
  className="css-jyjmas"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={false}
      disabled={true}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-umqk88"
      color="default"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={false}
      disabled={true}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-h1wp49"
      color="primary"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={false}
      disabled={true}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-49iodi"
      color="secondary"
    />
  </label>
</div>
`;

exports[`Storyshots Switch.State Uncontrolled 1`] = `
<div
  className="css-jyjmas"
>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-umqk88"
      color="default"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-h1wp49"
      color="primary"
    />
  </label>
  <label
    className="css-1tlrdmb"
  >
    <input
      className="css-1rnm75w"
      defaultChecked={true}
      disabled={false}
      onChange={[Function]}
      type="checkbox"
    />
    <i
      className="css-49iodi"
      color="secondary"
    />
  </label>
</div>
`;
